core/ev-admin.php

```diff
@@ -204,52 +215,66 @@ function update_videos_handler() {
     // Handle the ajax request
     check_ajax_referer( 'ev_settings' );
 
-    // Get options once now for the helper functions
+    // /*
```

Comment:
no - those are commented-out comment-out marks.
they're easier to turn off and on that way, at least in my text editor (atom)
removing them now!